keyboards/ploopyco/ploopyco.c

```diff
@@ -41,6 +41,9 @@
 #    ifndef PLOOPY_DPI_DEFAULT
 #        define PLOOPY_DPI_DEFAULT 1
 #    endif
+#ifndef PLOOPY_DRAGSCROLL_MOMENTARY
+#    define PLOOPY_DRAGSCROLL_MOMENTARY 1
+#endif
```

Comment:
On a completely separate note from this PR. https://github.com/qmk/qmk_firmware/pull/22967 these changes changed how the sensitivity for scrolling is defined. It seems that the default is really sensitive.

Reply:
https://github.com/beaudrychase/qmk_firmware/blob/38d6e9a75a4cff4ae5bd32209fc0cecb83c6f23b/keyboards/ploopyco/madromys/madromys.c#L32 The old way scroll sensitivity was configured was through changing the dpi settings while drag scroll was enabled. The default dpi for Adept was 900 and the drag scroll dpi was 100. This means that if the `PLOOPY_DRAGSCROLL_DIVISOR` was set to 9 instead of 8, the sensitivity should be the same for most people compared to their old firmware.

Reply:
If anything, if we're adding a separate dragscroll keycode, then I'd opt for completely removing this define.  So just one keycode for momentary, and one for toggle, and leave that behavior hard coded.  It makes things a bit less confusing for users.